src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return BallColor.values()[new Random().nextInt(BallColor.values().length)].name();
```

Comment:
We can add field with type `Random` to our class, so we won't need to create an instance every time when method invoked

Reply:
I understand that the code becomes less understandable because of such constructions. But in typical errors it was said -  don`t create additional variables, if possible. Where is this fine line?

Reply:
Or is the problem that we will create, each time a new object?

Reply:
yes, the problem is that we create a new object for each method call